src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -1268,7 +1275,8 @@ mobile_apis::VideoStreamingState::eType StateControllerImpl::CalcVideoState(
     ApplicationSharedPtr app,
     const mobile_apis::HMILevel::eType hmi_level) const {
   auto state = mobile_apis::VideoStreamingState::NOT_STREAMABLE;
-  if (IsStreamableHMILevel(hmi_level) && app->IsVideoApplication()) {
+
+  if (app->IsVideoApplication() && IsStreamableHMILevel(hmi_level)) {
```

Comment:
What is the reason of order change?

Reply:
@LuxoftAKutsan Just convenience and readability. If it is not Video app -> there is now sense to call the method which will determine if HMI level is streamable or not.